ui/src/components/overview/MobileClientCardView.js

```diff
@@ -47,7 +39,7 @@ class MobileClientCardView extends Component {
       <EmptyState>
         <EmptyStateTitle>{this.emptyStateMessage.noAppsCreated}</EmptyStateTitle>
         <EmptyStateAction>
-          <CreateClient createButtonSize="large" platforms={SUPPORTED_PLATFORMS} item={new MobileApp().toJSON()} />
+          <CreateClient createButtonSize="large" item={new MobileApp().toJSON()} />
```

Comment:
As suggested in another PR by @wei-lee, change the `item` to be a defaulted to `new MobileApp().toJSON()` instead of passing it here

Reply:
Let me address that later.

Reply:
This commit fixes this: https://github.com/aerogear/mobile-developer-console/pull/124/commits/35e9d0b4f4b4b56775bdf3f25df1de7d8e54bd82